sdk/containerregistry/Azure.Containers.ContainerRegistry/src/Blob/OciBlobDescriptor.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Security.Cryptography;
+using System.Text;
+using Azure.Core;
+
+namespace Azure.Containers.ContainerRegistry.Specialized
+{
+    [CodeGenModel("Descriptor")]
+    public partial class OciBlobDescriptor
+    {
+        /// <summary> Layer media type. </summary>
+        public string MediaType { get; set; }
+        /// <summary> Layer size. </summary>
+        public long? Size { get; set; }
```

Comment:
Is there any unit of measure for this size? ex. SizeInBytes

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/23833